File: web/src/api/yunguan/config/alertInfo.js
import request from '@/utils/request'
// 添加元素
export function deleteSystemAlert(modelData) {
    return request({
        url: 'sys/alert/deleteSystemAlert',
        method: 'post',
        data: modelData
    })
}
// 添加元素
export function updateSystemAlert(modelData) {
    return request({
        url: 'sys/alert/updateSystemAlert',
        method: 'post',
        data: modelData
    })
}
// 添加元素
export function getSystemAlertList(modelData) {
    return request({
        url: 'sys/alert/getSystemAlertList',
        method: 'post',
        data: modelData
    })
}

File: server/router/yunguan/system/alertInfo.go
package system

import (
	"github.com/gin-gonic/gin"
	v1 "ygpt/server/api/v1"
)

type SystemAlertRouter struct{}

func (s *SystemAlertRouter) InitSystemAlertRouter(Router *gin.RouterGroup) {
	dataRouter := Router.Group("sys/alert")
	modelApi := v1.ApiGroupApp.YunGuanApiGroup.SystemAlertApi
	{
		dataRouter.POST("addSystemAlert", modelApi.AddSystemAlert)         // 添加数据
		dataRouter.POST("deleteSystemAlert", modelApi.DeleteSystemAlert)   // 删除数据
		dataRouter.POST("updateSystemAlert", modelApi.UpdateSystemAlert)   // 删除数据
		dataRouter.POST("getSystemAlertList", modelApi.GetSystemAlertList) // 获取数据列表
	}
}
